docs/userguide/processing/baseline.py

```diff
@@ -276,6 +588,34 @@
 )
 blc.show_regions(axes["axe21"])
 
+# %% [markdown]
+# ### Widget for "advanced" baseline corrections
+# The `BaselineCorrector()` widget can be used in jupyter notebook (not jupyter lab).
+# - The buttons are as follows:
+#   - `upload`: upload files (disabled if a NDDataset is passed as parameter). Uploading file will not trigger the reading
+# and processing. To do so, the user is expected to click the `process` button.
+#   - `process`: baseline correct and plot original dataset + baseline and  corrected datasets
+#   - `save as`: save the baseline corrected NDDataset
+# - The `x slice` anbd `y slice` textboxes can be used to slice the initial dataset with the usual `[start:stop:step]`
+# format. In the `x` dimension, coordinates or indexes can be used (e.g. `[3000.0:2000.0:1]` or `[0:100:1]` are valid
+# entries). In the `y` dimension only indexes can be used (e.g. `[0:10:1]`). Note also that currently none of the
+# `start`, `stop`, `step` parameters can be omitted, e.g. `[3000.0:2000.0]` or `[:,:]` are not valid entries.
```

Comment:
This is not True anymore: you can enter as in numpy:
`[start:stop:step]`
`[:stop:step]`
`[::step]`
`[start::step]`
etc...

and `start:` etc.... without the bracket

Reply:
and this for x and y, and also you can use index or location for both.